subt_ign/src/GameLogicPlugin.cc

```diff
@@ -458,6 +461,9 @@ void GameLogicPlugin::Configure(const ignition::gazebo::Entity & /*_entity*/,
   this->dataPtr->startPub =
     this->dataPtr->node.Advertise<ignition::msgs::StringMsg>("/subt/start");
 
+  this->dataPtr->competitionClockPub =
+    this->dataPtr->node.Advertise<ignition::msgs::Clock>("/subt/clock");
```

Comment:
Consider changing the topic name to indicate that this is a count down clock so as to avoid confusion with `/clock`.

Reply:
Changed to `run_clock` 607c6de91d9baa5483bb824e18ba42aae172b934